### DISCUSSION TITLE
Resume session between local and remote access?

### DISCUSSION THREAD

**Discussion by shigutso** [neutrinolabs/xrdp#2203] [Q&A]
Hi, sorry if this was already answered but I could not find anything about it
At the current state of this software, is it possible to login locally to a Linux desktop, lock the screen and, from another PC (Windows), remote connect to it and resume the session, with the screen still locked (or at least blank) at the local side?

If not, is this a planned feature?
Thanks!

**Comment by matt335672** [comment]
Hi @shigutso 

To answer your specific question - "no it isn't". The best you can do is run an RDP session, and connect to it on the console using an RDP client. At the moment even that won't work quite as you want, as we haven't got the session handover working properly yet. Expect that in the next major release.

Have a look at #2122 which should give you a better idea on how the architecture works, and what is currently possible. I can't see an easy way to get exactly what you want into the product I'm afraid.
